Resources/Prototypes/GameRules/roundstart.yml

```
@@ -102,9 +102,8 @@
       - type: NpcFactionMember
         factions:
         - Syndicate
-      mindComponents:
-      - type: NukeopsRole
-        prototype: NukeopsCommander
+      mindRoles:
+      - MindRoleNukeopsCommander
```

YAMLlint (1.35.1)

[error] wrong indentation: expected at least 7

(indentation)